.github/workflows/ci-delta-core.yml

```diff
@@ -23,11 +28,8 @@ jobs:
           sbt -Dsbt.color=always -Dsbt.supershell=false \
             "project delta" \
             clean \
-            scalafmtCheck \
-            Test/scalafmtCheck \
-            scalafmtSbtCheck \
-            scapegoat \
-            doc
+            core-static-analysis
```

Comment:
The `core-static-analysis` should run also for plugins here. But it can't just be added to the `coreModules` as is now because that would also run the unit tests for `plugins` through `core-unit-tests-with-coverage`

Reply:
Perhaps we can add it to the plugins build instead?

Reply:
That should work yes